backend/src/main.ts

```diff
@@ -345,21 +383,36 @@ export const moodLog = async (req: UserRequest, res: Response) => {
             logData.audio = "inprogress";
         }
 
+        const lastUpdated = (
+          await db.ref(`/${req.user!.user_id}/lastUpdated`).get()
+        ).val();
         if (data.addFlag && data.addFlag.startsWith("summary:")) {
             logData.time = "12:00 PM";
             logData.zone = "local";
             logData.addFlag = "summary";
             logData.timeLogged = DateTime.utc().toMillis();
 
-            const lastUpdated = (await db.ref(`/${req.user!.user_id}/lastUpdated`).get()).val();
             if (lastUpdated && lastUpdated > globalNow.toMillis()) {
                 setLastUpdated = false;
                 promises.push(db.ref(`/${req.user!.user_id}/offline`).set(Math.random()));
             }
         }
+
+        if (data.addFlag && data.addFlag.includes("offlineSync:")) {
+            console.log("Adding offline sync flag");
+            if (lastUpdated && lastUpdated > globalNow.toMillis()) {
+                setLastUpdated = false;
+            }
+            promises.push(
+                db.ref(`/${req.user!.user_id}/offline`).set(Math.random())
+            );
+        }
+
+        // maybe use addFlag to determine if unsynced + update offline
```

Comment:
?

Reply:
didn't u say this is necessary? resent the offline number?